### PR TITLE
Add a basic model for Known Exploited Vulnerabilities 

### PR DESCRIPTION
**Pull request by ziadhany**
image
Issue #1028

### CODE REVIEW

**Comment by TG1999** [comment]
I think we need a new tab for this, @DennisClark what's your initial thoughts on this ?

File: vulnerabilities/improvers/vulnerability_kev.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+
+class VulnerabilityKevImprover(Improver):
+    """
+    Known Exploited Vulnerabilities Improver
+    """
+
+    @property
+    def interesting_advisories(self) -> QuerySet:
+        return [Advisory.objects.all()[0]]
```
Comment (ziadhany) [inline vulnerabilities/improvers/vulnerability_kev.py:26]: I am not sure if this is the right way to just run the improver

File: vulnerabilities/improvers/vulnerability_kev.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            logger.error(
+                f"Failed to fetch the CISA Catalog of Known Exploited Vulnerabilities: {kev_url}"
+            )
+            return []
+        kev_vulnerabilities = kev_data["vulnerabilities"]
+        for kev_vul in kev_vulnerabilities:
+            alias = Alias.objects.get_or_none(alias=kev_vul["cveID"])
+            if not alias:
+                continue
+
```
Comment (keshav-space) [inline vulnerabilities/improvers/vulnerability_kev.py:42]: Add a `TODO` to revisit this once we have a better, improved design. The current `Improver` design is constraining.

**Review by keshav-space — CHANGES_REQUESTED** [review]
@ziadhany Thanks ++ 
Looking good! see minor nits for your consideration.
Also, please add some tests.

File: vulnerabilities/improvers/vulnerability_kev.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+class VulnerabilityKevImprover(Improver):
+    """
+    Known Exploited Vulnerabilities Improver
+    """
+
+    @property
+    def interesting_advisories(self) -> QuerySet:
+        return [Advisory.objects.all()[0]]
```
[the remark this answers is not in the document]
Comment (ziadhany) [reply vulnerabilities/improvers/vulnerability_kev.py]: Done

File: vulnerabilities/improvers/vulnerability_kev.py (hunk 2 above)
Comment (ziadhany) [reply vulnerabilities/improvers/vulnerability_kev.py]: Done

**Review by keshav-space — APPROVED** [review]
Thanks @ziadhany, Merging!

**Pull request merged**